lib/pure/json.nim

```diff
@@ -353,6 +353,14 @@ proc `[]=`*(obj: JsonNode, key: string, val: JsonNode) {.inline.} =
   assert(obj.kind == JObject)
   obj.fields[key] = val
 
+proc `%`*(table: StringTableRef): JsonNode =
```

Comment:
Shouldn't this be done inside `strtabs` itself? The module can make use of external `%` implementation afaik (and if it doesn't, we can make it do so)

Reply:
Well the json module already contains the implementation for `Table` and `OrderedTable` so it just felt more at home along with them. But I don't object to moving it.

Reply:
Well it does save us from having to export the `mode` field :P

Reply:
True, but at least this field should be readable through a template anyways..